### DISCUSSION TITLE
3 requests.

### DISCUSSION THREAD

**Discussion by doo158** [MaxRobinsonTheGreat/EvolutionSimulatorV2#29] [Ideas and Suggestions]
1. the ability to change the probability of different cell types evolving. for instance, if you really loved mover cells, you'd be able to raise the chance of them evolving, using a parameter in the simulation controls tab. if you really hated eyes, you'd be able to jack the probability for eyes all the way down to 0, effectively disabling them. perhaps there could also be a checkbox similar to the "use evolved mutation rate" button, which would let you toggle between all creatures using your universal settings, or being able to use their own mutation rates for certain parts.

2. imagine that producers generate food using some external energy source. one feature that i would like to see is the ability to change this kind of energy source, like giving it a different intensity in different areas, or making it fluctuate with time. the intensity of the production rate of the plants would be directly proportional to the downpour of energy. once again, there could be a checkbox to toggle between the universal production rate, or your custom one.

3. the ability to save and load creatures, worlds, and settings. perhaps there could be an extra tab, dedicated to saving and loading different things. there could be 3 different categories. one for creatures, one for worlds, and one for settings. it would definitely lift the hassle of having to reset your settings every time you reopen the game, or rebuilding a creature from scratch if you wanted to use it again.

**Comment by xkortex** [comment]
I like the idea of 2. Like some sort of seasonal variation. 

3 is something I've been definitely feeling the need for. I think that is worth of its own thread.
